script.js

```diff
@@ -150,6 +156,22 @@ document.addEventListener('keydown', function(e) {
         }
         e.preventDefault();
     }
+
+    if (isAltKey && isEnter) {
+        skipButton.click();
+        e.preventDefault();
+    }
+
+    if (isEsc) {
+        const globalPopup = document.querySelector('.global-popup');
+        if (!globalPopup || globalPopup.style.display === "none") {
+            interruptButton.click();
+            e.preventDefault();
+        } else {
+            if (!globalPopup) return;
+            globalPopup.style.display = "none";
```

Comment:
This `else` isn't necessary, the existing handler is still gonna fire.

Esc is also used to close the [full screen image viewer](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/4afaaf8a020c1df457bcf7250cb1c7f609699fa7/javascript/imageviewer.js#L116-L117).

Reply:
Thanks for the tip! Made the edits.